src/dcat-ap/dcat-dataset.test.ts

```diff
@@ -1,15 +1,15 @@
 import { cloneObject } from '@esri/hub-common';
 import { DcatDataset } from './dcat-dataset';
 
-import * as datasetFromApi from './test-helpers/mock-dataset.json';
+import * as datasetFromApi from '../test-helpers/mock-dataset.json';
 
 const siteUrl = 'https://foobar.hub.arcgis.com'
 const orgTitle = 'My Fun Org'
 const portalUrl = 'https://my-fun-org.maps.arcgis.com'
 
 describe('DcatDataset', () => {
   it('Dataset props come from right places', function() {
-    const dataset = new DcatDataset(datasetFromApi.attributes, portalUrl, orgTitle, siteUrl)
```

Comment:
I thought we got the whole response back from Mark's `searchContent` Hub.js function, but we actually just get the attributes.